bcos-ledger/src/libledger/Ledger.cpp

```diff
@@ -268,69 +270,31 @@ void Ledger::asyncPrewriteBlock(bcos::storage::StorageInterface::Ptr storage,
         LEDGER_LOG(WARNING) << "Empty transactions and metadata, empty block?"
                             << LOG_KV("blockNumber", blockNumberStr);
     }
-    bytes transactionsBuffer;
-    transactionsBlock->encode(transactionsBuffer);
 
-    Entry number2TransactionHashesEntry;
-    number2TransactionHashesEntry.importFields({std::move(transactionsBuffer)});
-    storage->asyncSetRow(SYS_NUMBER_2_TXS, blockNumberStr, std::move(number2TransactionHashesEntry),
-        [setRowCallback](auto&& error) { setRowCallback(std::forward<decltype(error)>(error)); });
+    if (blockVersion < uint32_t(bcos::protocol::BlockVersion::V3_6_VERSION))
```

Comment:
判断版本号，统一用大于等于，不会留有边界
```suggestion
    if (blockVersion >= uint32_t(bcos::protocol::BlockVersion::V3_6_VERSION)) 
    {
        // do nothing
    }
    else
    {
```